### ISSUE TITLE
Issue: Update project list

### ISSUE THREAD

**Issue by maudbv** [BiologicalRecordsCentre/ABLE#693]
### Describe the issue

The current list of projects in the Beta version needs to be updated as it is only a "test" list.

### Expected behaviour

The project list should include at least the following: 
- **Agroecology-TRANSECT** (valid for all European countries. I don’t think we need to use countries as a sub-project name, as this is predefined separately)
- **CAP4GI**    (Germany only)
- **Urban Nature Plans + (UNP+)**     (note that the project name has changed from UGP+ to UNP+; all European countries)
- **VielFalterGarten**  (Germany only)
- **15-minute Protocol testing (TMD)**   (Germany only)

### App Version

1.26.0 (344)

### Device

iPhone SE

### When did this happen

19.06.2024

**Comment by kazlauskis** [comment]
@johnvanbreda is there something on the website that would let users delete and/or edit their projects?

**Comment by johnvanbreda** [comment]
Yes, they can see their projects at https://butterfly-monitoring.net/projects and access an edit link for the ones they are admin of (e.g. the ones they created).

**Comment by CrisSevilleja** [comment]
This project page should be on the website under one of the tabs. @DavidRoy do you think we should make this project page visible to everyone or how we should be present?

**Comment by DavidRoy** [comment]
Agreed. Which menu do you suggest?

**Comment by CrisSevilleja** [comment]
not sure... If the page should be visible for everyone, all users, it should be under My Data tab. But if it is only for coordinators under Scheme adm. tab

**Comment by DavidRoy** [comment]
main issue resolved
